cloud-run/fhir-converter/FHIR-Converter-Installation-And-Usage-Guide.md

```diff
@@ -1,6 +1,22 @@
 # Microsoft FHIR Converter CLI Installation Guide
 This document provides a guide for installing the [Microsoft FHIR Converter](https://github.com/microsoft/FHIR-Converter) as a Command Line Interface (CLI) tool on Windows, MacOS, and Linux systems, as well as a brief introduction to using the converter.
 
+## Running via Docker
+To run the FHIR Converter in a Docker container, follow these steps:  
+  
+  1. From the `cloud-run/fhir-converter` directory, run:  
+  `docker build -t fhir-converter .`
+  1. Start a container by running:  
+  `docker run -p 8080:8080 fhir-converter`
+  1. Using curl or a REST client like Postman or Insomnia, make a POST request to http://localhost:8080/convert-to-fhir with a request body:
+  ```
+  {
+    "input_data": "VALID_INPUT_DATA",
+    "input_type": "hl7v2",
+    "root_template": "ADT_A01",
+  }
+  ```
```

Comment:
Major nit, but could make it clear that this is an example request. Calling `VALID_INPUT_DATA` does this in part, but I'm concerned a reader might assume that `"input_data"` must be "hl7v2" and that `"root_template"` must be "ADT_A01".

Reply:
Good call, will make this more clear